### PR TITLE
docs: introduced appearance articles from form, pager, toolbar

### PR DESCRIPTION
**Pull request by yanisslav**

### CODE REVIEW

File: components/form/appearance.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <TelerikForm Model="@TestUser"
+                         Size="@size">
+                <FormItems>
+                    <FormItem Field="@nameof(User.FirstName)"></FormItem>
+                    <FormItem Field="@nameof(User.LastName)"></FormItem>
+                </FormItems>
+            </TelerikForm>
+        </div>
+    }
+    @ code{
```
Comment (svdimitr) [inline components/form/appearance.md:48]: This snippet will not compile, can you remove the space between `@` and `code`

File: components/form/appearance.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the form by setting the [Size](#size) attribute.
+
+## Size
+
+You can increase or decrease the size of the Form by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Form.Size` class:
```
Comment (svdimitr) [inline components/form/appearance.md:17]: Maybe you can add some more information that the Size controls the padding for all form editors and buttons.

File: components/form/appearance.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+| Class members | Manual declarations |
+|---------------|--------|
+| `Small`   |`sm`|
+| `Medium`   |`md`|
```
Comment (svdimitr) [inline components/form/appearance.md:22]: Add that `Medium` is the default size option.

Applies to all articles.

Comment (svdimitr) [inline components/form/appearance.md:22]: Add that `Medium` is the default size option.

Applies to all articles.

File: components/form/appearance.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+| `Large`   |`lg`|
+
+If the size option is not specified, it is set to `Medium` by default.
```
Comment (ntacheva) [inline components/form/appearance.md:27]: Look how the default options is specified in other appearance articles - e.g. https://docs.telerik.com/blazor-ui/components/autocomplete/appearance#fillmode.

Listed in the table for better visibility. Let's keep that approach for consistency. Applies to all articles.

File: components/form/appearance.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the form by setting the [Size](#size) attribute.
+
+## Size
+
+>The `Size` configuration of the Form affects the padding of all the editors and buttons.
```
Comment (ntacheva) [inline components/form/appearance.md:17]: Move that not after the first paragraph to provide the main information for the parameter first and then list the specifics.

**Review by ntacheva — CHANGES_REQUESTED** [review]
Listed in the comments above.

**Comment by ntacheva** [comment]
Link the "Appearance" articles in the "Overview" of the corresponding components. Reference: http://localhost/blazor-ui/components/autocomplete/overview#styling-and-appearance

File: components/form/overview.md [hunk 6]
```diff
@@ -255,6 +255,16 @@ Use the Form reference to get access to its `EditContext`. The Form generates th
 }
 ````
 
+## Styling and Appearance
```
Comment (ntacheva) [inline components/form/overview.md:258]: Look how this section is styled and positioned in other components' "Overview" articles. Let's keep the same formatting and position for consistency. Applies to all articles.

**Review by ntacheva — CHANGES_REQUESTED** [review]
Apologies for not mentioning this earlier - we need to link the demo pages from the "See also" section in all three articles.

File: components/form/overview.md [hunk 7]
```diff
@@ -212,6 +212,16 @@ The Blazor Form exposes multiple parameters that allow you to customize its layo
 | `ColumnSpacing`  | `string` | Defines the amout of vertical space between the Columns. See the [Columns]({%slug form-columns%}) article for more information. |
 | `Orientation`  | `FormOrientation` enum <br /> (`Vertical`) | controls the orientation of the Form. See the [Orientation]({%slug form-orientation%}) article for more information. |
 
+## Styling and Appearance
```
Comment (ntacheva) [inline components/form/overview.md:215]:
```suggestion
### Styling and Appearance
```

File: components/toolbar/overview.md [hunk 8]
```diff
@@ -85,6 +85,16 @@ The Blazor ToolBar provides parameters to configure the component:
 | `Adaptive ` | `bool` | Toggles the overflow popup of the toolbar. Displays an additional anchor on the right of the toolbar, where it places all items which overflow from the toolbar. |
 | `Class` | `string` | The CSS class to be rendered on the main wrapping element of the ToolBar component, which is `<div class="k-toolbar">`. Use for [styling customizations]({%slug themes-override%}). |
 
+## Styling and Appearance
```
Comment (ntacheva) [inline components/toolbar/overview.md:88]:
```suggestion
### Styling and Appearance
```

**Review by ntacheva — APPROVED** [review]

**Pull request merged**